env.sh

```diff
@@ -8,25 +8,27 @@ if [ -z "$PODIO" ]; then
 fi
 
 unamestr=`uname`
-if [[ "$unamestr" == 'Linux' ]]; then
-	echo $LD_LIBRARY_PATH | grep $PODIO/lib >& /dev/null
-	if [ $? == "1" ]; then
+if [[ "$unamestr" = 'Linux' ]]; then
+  if [ "$(echo $LD_LIBRARY_PATH | grep -o $PODIO/lib)" = "" ]; then
```

Comment:
```suggestion
  if ! echo $LD_LIBRARY_PATH | grep $PODIO/lib > /dev/null 2>&1; then
```
I think this is the most idiomatic (POSIX compliant) one liner to check this. It uses the non-zero exit code of `grep` if it doesn't find a match, and we simply redirect all output to `/dev/null` to not clutter the output. Also avoids creating a subshell.